.travis.yml

```diff
@@ -57,22 +57,21 @@ jobs:
 
     - name: "Code imports"
       before_install: skip
-      install:
-        - pip install isort
+      install: pip install -r requirements/dev.txt
```

Comment:
this is intentional to keep up to date with the latest version(s) of isort, please revert

Reply:
https://github.com/open-zaak/open-zaak/issues/638#issue-651432502:
> **Describe the bug**
> Currently 'isort`(v4) fails due to changed command line parameters. We should upgrade (and pin in requirements - not the travis job)`isort` to v5 and update all code to work with that version.
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
>     1. Run `isort` (v5) with command line params as specified in Travis job.
> 
>     2. See failures
> 
> 
> **Expected behavior**
> Should just work.

Joeri specifically asked me to pin them in the requirements and not to install in the travis job